File: exercise/sentinel2.py
userinput = int(input("What is your value: "))

sumed = 0
count = 0
senitel = 0

while userinput > senitel:
    
    sumed+=userinput

    count+=1

    userinput = int(input("Input another value or enter 0 to see sum of all values enterd: "))

mean = sumed/count

print ("\nYour sumed value is ==> ",sumed)
print ("\nThe mean of all numbers is ==> ",mean, "\n")
    

File: exercise/sentinel2.c
#include <stdio.h>

int sumed;
int input;
int mean;
int senitel = 0;
int count = 0;
int main(){

    printf("What is your value: ");
    scanf("%d", &input);

    while(input > senitel){
       
        sumed+=input;
        
        printf("Input another number or a value greater than 0: ");
        scanf("%d", &input);
        
        count+=1;

    }

    mean = sumed/count;

    printf("\nYour sumed value is ==> %d",sumed);
    printf("\nThe mean of all numbers is ==> %d",mean);
}
